### PR TITLE
Add unmaintained notice

### PR DESCRIPTION
**Pull request by charlax**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 
 .. image:: https://coveralls.io/repos/uber/charlatan/badge.png
   :target: https://coveralls.io/r/uber/charlatan
 
-.. image:: https://pypip.in/version/charlatan/badge.svg
-    :target: https://pypi.python.org/pypi/charlatan/
-    :alt: Latest Version
-
-**Efficiently manage and install data fixtures**
+**⚠️ This repository is not actively maintained anymore.**
```
Comment (carolinevdh) [inline README.rst]: Has this emoji push been tested (((:

File: README.rst [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
+**Efficiently manage and install data fixtures**
 
 .. image:: https://travis-ci.org/uber/charlatan.png?branch=master
     :target: https://travis-ci.org/uber/charlatan
 
 .. image:: https://coveralls.io/repos/uber/charlatan/badge.png
   :target: https://coveralls.io/r/uber/charlatan
 
-.. image:: https://pypip.in/version/charlatan/badge.svg
-    :target: https://pypi.python.org/pypi/charlatan/
```
Comment (carolinevdh) [inline README.rst]: Why remove the pypi link?

Comment (charlax) [inline README.rst]: It's broken.

File: README.rst (hunk 1 above)
Comment (charlax) [inline README.rst]: :)

**Pull request merged**